### DISCUSSION TITLE
why legacy-API instead Options-API?

### DISCUSSION THREAD

**Discussion by laoshaw** [intlify/vue-i18n#860] [General]
Legacy API means it's going to be obsolete to me, but I think vue3 had no plan to phase out options-API anytime soon(vue3 options-api is more beginner friendly, plus not everyone does complex SPA with hundreds of components that 'requires' composition API').

**Comment by kazupon** [comment]
When the composition API was first released, it was said that the options API would replace the composition API in Vue 3 later versions (maybe Vue 4?).

Currently, vue-i18n-next supports both the legacy API, which is similar in style to the options API and the composition API.
The legacy API is not immediately deprecated at this time. In the future, when it will be deprecated, we will plan to deprecate the legacy API at the same time.
